modelforge/train/training.py

```diff
@@ -147,6 +151,10 @@ def __init__(self, requested_properties: List[str]):
         self.requested_properties = requested_properties
         self.include_force = "per_atom_force" in self.requested_properties
         self.include_charges = "per_system_total_charge" in self.requested_properties
+        # dipole_moment is calculated from charges, thus if dipole moment is requested
+        # we also need to calculate charges, even if we don't call per_system_total_charge
+        if "per_system_dipole_moment" in self.requested_properties:
```

Comment:
Need documentation with the dependency of properties, probably together with the list of properties available.

Reply:
Same as comment above; we need to have this in documentation. Good idea.